compute/instances/api-cli/migrating-instances.mdx

```diff
@@ -1,213 +1,110 @@
+  - The Instance cannot be in a placement group
+</Message>
+
+1. Stop the Instance. Replace `{zone}` with the Availability Zone of your Instance. This value can be `fr-par-1` for Instances located in Paris, for example. Replace `{server_id}` with the ID of your Instance.
 
     Request:
     ```
-    curl -H "X-Auth-Token: $SECRET_KEY" https://api.scaleway.com/instance/v1/zones/<REGION>/servers/<INSTANCE_ID>/action -d '{"action" : "backup"}' -H "Content-Type: application/json"
+    curl -X PATCH -H "X-Auth-Token: $SCW_SECRET_KEY" -H "Content-Type: application/json" -d '{"action" : "poweroff"}' "https://api.scaleway.com/instance/v1/zones/{zone}/servers/{server_id}/action"
     ```
 
     Return:
     ```
     {
-        "task": {
-            "description": "server_backup",
-            "href_from": "/servers/13d0c8d4-cc56-4f8b-b15c-ea25e8dbe989/action",
-            "href_result": "/images/88996119-edb9-4283-8a6e-f9b98999bc19",
-            "id": "b209f550-c8e1-4378-83e7-961dd834eb9c",
-            "progress": 0,
-            "started_at": "2019-05-14T13:14:01.573950+00:00",
-            "status": "pending",
-            "terminated_at": null
-        }
+    "task":{
+        "id":"cd1cf3da-6dee-4c96-bc86-f4fe848f36d4",
+        "description":"server_poweroff",
+        "status":"pending",
+        "href_from":"/servers/85bde2188-1657-11ee-be56-0242ac120002/action",
+        "href_result":"/servers/85bde2188-1657-11ee-be56-0242ac120002",
+        "started_at":"2023-06-29T08:20:36.927830+00:00",
+        "terminated_at":null
+    }
+    }
     ```
 
-    The ID of the image is indicated in the line `"href_result"`: `"/images/88996119-edb9-4283-8a6e-f9b98999bc19"` Take a note of the ID, as it will be required in the next step.
-2. Create the new instance based on the backup created previously. To do so, use the image ID from the `"href_result"`.
+2. Update the `commercial_type` of your Instance.
 
     Request:
     ```
-    curl https://api.scaleway.com/instance/v1/zones/<REGION>/servers \
-    -H "X-Auth-Token: $SECRET_KEY" \
-    -H "Content-Type: application/json" \
-    -d '{ \
-    "name": "<NEW_INSTANCE_NAME>",
-    "image": "88996119-edb9-4283-8a6e-f9b98999bc19",
-    "commercial_type": "GP1-S",
-    "tags": ["migration"],
-    "organization": "<ORGANIZATION_ID>"}'
+    curl -X PATCH -H "X-Auth-Token: $SCW_SECRET_KEY" -H "Content-Type: application/json" -d '{"commercial_type" : "<NEW_INSTANCE_TYPE>"}' "https://api.scaleway.com/instance/v1/zones/{zone}/servers/{server_id}"
     ```
+    <Message type="important">
+        Local storage constraints must be respected. For example, downgrading a DEV1-L instance with 80GB of local SSD storage to a DEV1-S instance is not possible because the DEV1-S offer only supports a maximum of 20GB. However, if the DEV1-L instance is only using block storage (`b_ssd`) without relying on local SSD storage, the downgrade can be performed without any issues.
```

Comment:
```suggestion
        Local storage constraints must be respected. For example, downgrading a DEV1-L Instance with 80GB of local SSD storage to a DEV1-S Instance is not possible because the DEV1-S offer only supports a maximum of 20GB. However, if the DEV1-L Instance is only using block storage (`b_ssd`) without relying on local SSD storage, the downgrade can be performed without any issues.
```